java/pom.xml

```diff
@@ -28,18 +28,21 @@
   <url>https://arrow.apache.org/</url>
 
   <properties>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
     <target.gen.source.path>${project.build.directory}/generated-sources</target.gen.source.path>
     <dep.junit.platform.version>1.4.0</dep.junit.platform.version>
     <dep.junit.jupiter.version>5.4.0</dep.junit.jupiter.version>
-    <dep.slf4j.version>1.7.25</dep.slf4j.version>
+    <dep.slf4j.version>1.7.36</dep.slf4j.version>
     <dep.guava.version>30.1.1-jre</dep.guava.version>
-    <dep.netty.version>4.1.72.Final</dep.netty.version>
     <dep.jackson-bom.version>2.13.2.20220328</dep.jackson-bom.version>
+    <dep.netty.version>4.1.76.Final</dep.netty.version>
+    <dep.netty-bom.version>4.1.76.Final</dep.netty-bom.version>
     <dep.hadoop.version>2.7.1</dep.hadoop.version>
     <dep.fbs.version>1.12.0</dep.fbs.version>
     <dep.avro.version>1.10.0</dep.avro.version>
     <arrow.vector.classifier />
-    <forkCount>2</forkCount>
+    <forkCount>0</forkCount>
```

Comment:
Is this related to JPMS ?

Reply:
To be reviewed. Value more than "0" produce: NoClassDefFound Could not initialize class org.apache.arrow.memory.RootAllocator

Reply:
Updated to 2